src/main/resources/application.yml

```diff
@@ -89,7 +89,7 @@ data-grid:
 # the thread that sends actions to handlers
 action-distribution:
   # how many actions should we read each time we wake
-  retrieval-max: 200
+  retrieval-max: 1
```

Comment:
? seems like an accidental change.

Reply:
Yep was accidentally left there after testing the locking mechanism in postgres. Will revert